database.py

```diff
@@ -144,7 +144,7 @@ def get_dark(self, threshold=0.6, all_pkmn=False):
     def __get_region(self, region):
         # Helper method for getting all the Pokemon of a specified region.
         return [pokemon for pokemon in self.__pokemon_list
-                if pokemon.get_region() == region]
+                if pokemon.get_region() == region and not pokemon.is_extra()]
```

Comment:
As you can see here I explicitly excluded extra pokemon from the `get_region` methods.